### DISCUSSION TITLE
Raising timeout limit?

### DISCUSSION THREAD

**Discussion by suntong** [deta/cloud-docs#558] [General] [answered]
I have a vague memory of seeing a page where you can request to raise timeout limit, is that so?

Anyway, is it possible that I can request to raise timeout limit to one of my micro please? It's not CPU intensive but network bound so need much longer time to overcome the network delay.

**Comment by lemonyte** [comment, marked as answer]
[Docs](https://docs.deta.sh/docs/micros/faqs_micros)

[Timeout Request Form](https://form.deta.dev/timeout)
